app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt

```diff
@@ -29,7 +27,15 @@ class ProfileListAdapter(
     ListAdapter<Profile, ProfileListAdapter.ViewHolder>(
         ProfileDiffCallbacks()
     ) {
-
+    companion object {
+        const val TAG = "SimpleDialog"
+        fun newInstance() = DetailsFragment().apply {
+            val args = Bundle()
```

Comment:
Take the args variable as the function parameter.

Reply:
@rob729  Took the args variable as function parameter, as you suggested